README.md

```diff
@@ -9,7 +9,7 @@
 | Spekt.VSTest.Coverage.Report | [![NuGet](https://img.shields.io/nuget/v/Spekt.VSTest.Coverage.Report.svg)](https://www.nuget.org/packages/Spekt.VSTest.Coverage.Report/) | [![MyGet Pre Release](https://img.shields.io/myget/spekt/vpre/Spekt.VSTest.Coverage.Report.svg)](https://www.myget.org/feed/spekt/package/nuget/Spekt.VSTest.Coverage.Report) |
 
 ## Vision
-Stop .NET(any) developer from shipping a bug. [software failures cost the worldwide economy $1.1 trillion in 2016](https://crossbrowsertesting.com/blog/development/software-bug-cost/).
+`Stop .NET(any) developer from shipping a bug`. If we have to say nicely `Let's not ship another bug` because [software failures cost the worldwide economy $1.1 trillion in 2016](https://crossbrowsertesting.com/blog/development/software-bug-cost/) and everybody hates bug.
```

Comment:
It may be a bit broad.

I'd word it like - "... with continuous monitoring of software quality using code coverage as a metric on any editor".

Reply:
>It may be a bit broad.
Agreed.
>I'd word it like - "... with continuous monitoring of software quality using code coverage as a metric on any editor".

Impress me with your contribution to this repo. I will consider your request 😄

Reply:
I thought it was (2) here: https://github.com/spekt/vstest.coverage/graphs/contributors

Reply:
@singhsarab @jayaranigarg @ShreyasRmsft you guys have been right throughout. Let's felicitate Mr Madala once this hackathon is over.

Reply:
>@singhsarab @jayaranigarg @ShreyasRmsft you guys have been right throughout. Let's felicitate Mr Madala once this hackathon is over.

I appreciate this idea 😄